sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationClientTest.java

```diff
@@ -537,7 +539,7 @@ public void listSettingsWithPagination() {
         }
 
         SettingSelector filter = new SettingSelector().keys(keyPrefix + "-*").labels(labelPrefix);
-        assertEquals(numberExpected, ((List<ConfigurationSetting>) client.listSettings(filter)).size());
+        assertEquals(numberExpected, client.listSettings(filter).streamByPage().count());
```

Comment:
This doesn't seem right. I would have assumed this would be `stream()`, not `streamByPage()`.